Lib/threading.py

```diff
@@ -845,6 +846,10 @@ class is implemented.
         else:
             self._daemonic = current_thread().daemon
         self._ident = None
+        if context is not None:
+            self._context = context
+        else:
+            self._context = _copy_context()
```

Comment:
Is _copy_context() fast/efficient when contextvars is not used at all?

Reply:
This is information from [PEP-567](https://www.python.org/dev/peps/pep-0567/):
> The Context mapping is implemented using an immutable dictionary. This allows for a O(1) implementation of the copy_context() function. 

I believe it is fast/efficient to use `copy_context`.